### PR TITLE
Fixes for umayyads-actors

### PR DESCRIPTION
**Pull request by mrlie58**
- General (all Civilisations)

  - Improved actors (units, props)
    - add, change and remove some variants
    - correct indentation
  - correct {civ}.json files
  - correct shieldwall.xml
  - correct some maps

- Specific

  - Anglo-Saxons
    - correct actor storehouse.xml
    - correct siege_ram icon
  - Byzantines
    - change some unit and structures templates
    - correct siege_ram icon
  - Carolingians
    - change trebuchet-templates
    - correct siege_ram icon
  - Norse
    - change some unit and structures templates
  - Umayyads
    - attach coif prop
    - attach new head to female citizen

### CODE REVIEW

**Comment by Nescio0** [comment]
The purpose of the opening post is to inform people what the pull request does (and why it's an improvement). You don't have to list every individual file, you can group and summarize your changes, e.g.:
* Improved Umayyad unit actors:
  * add, change, and remove some variants;
  * attach coif prop;
  * correct indentation;
  * etc.

>Add map `umayyads.xml` with all umayyads-actors

`git mv umayyads.xml sandbox_umayyads.xml`

>Merge https://github.com/0ADMods/millenniumad/pull/54 into local repository

That will introduce a merge conflict.

**Comment by mrlie58** [comment]
> 
> 
> Some of the units previously used objectcolor, which is a perfectly valid material, why did you change it for player color?

I've researched the Main game and found `<material>objectcolor.xml</material>` not in unit-actors, but in "props" and some "trader_h". Therefore i only changed the norse and anglo actors (forgot `anglo_spearman_a` and `anglo_spearman_b` and leave the others as is.

**Comment by StanleySweet** [comment]
It can be used anywhere instead of using player color it uses the specified color

**Comment by StanleySweet** [comment]
Please revert the objectcolor changes.

**Comment by Nescio0** [comment]
And please undo your changes to the (six) templates, we don't want a merge conflict, nor regression.

Also rename the map you introduced:

`git mv umayyads.xml sandbox_umayyads.xml`

**Comment by mrlie58** [comment]
> 
> 
> Please revert the objectcolor changes.

DONE

**Comment by Nescio0** [comment]
> DONE

Still not done:

**Comment by mrlie58** [comment]
> 
> 
> > DONE
> 
> Still not done:
> 

I've really problems to correct this ;-) Any of my Editors seems to format the files i have to change. Only the good, old MS-Editor shows me the correct(wrong) indentation. So i will try to just copy an original file from an ealier downloaded milleniumad-mod over the actual file. But i'm afraid to give you more work as necessary. Is there a simplier way to fix this?

**Comment by Nescio0** [comment]
@mrlie58, you can do it manually via your webbrowser, by sign in to github, going to https://github.com/mrlie58/millenniumad, browse to find the file(s) you want to change, click on “Edit this file” (the pencil symbol), make the changes (or in this case, copy-and-paste the old lines to restore them), then commit the changes directly to your branch.
Because you do have to do the files one by one, this is really inefficient for larger pull requests. However, in this case it was just a few lines in two files.
As you can see I've already fix the two offending templates, because it was more work explaining than doing it myself. This patch no longer has a merge conflict, but you still have to address @StanleySweet's point, hence why I'm writing this explanation.

**Comment by mrlie58** [comment]
> 
> 
> @mrlie58, you can do it manually via your webbrowser:
> 
>     * signing in to github
> 
>     * go to https://github.com/mrlie58/millenniumad
> 
>     * browse to find the file(s) you want to change
> 
>     * click on “Edit this file” (the pencil symbol)
> 
>     * make the changes (or in this case, copy-and-paste the old lines to restore them)
> 
>     * then click “Commit the changes directly to the master branch” and “Commit changes”.
> 
> 
> Because you do have to do the files one by one, this is really inefficient for larger pull requests. However, in this case it was just a few lines in two files.
> As you can see I've already fixed the two offending templates, because it was more work explaining than doing it myself. This patch thus no longer has a merge conflict, but you still have to address @StanleySweet's point, hence why I'm writing this explanation for you nonetheless.
> 
> [EDIT] And edit your opening post (i.e. the first message in this conversation) to reflect everything it does, since it includes all your commits.

Related to Stan's point i've done this at the same time as yours, but i think i've made it wrong because i titled it "**adress requested changes (stan)**".

File: art/actors/units/byzantines/cavalry_archer_e_r.xml [hunk 1]
```diff
@@ -35,16 +35,9 @@
...
-		<variant file="biped/rider/attack_slaughter_shield_hele.xml"/>
-		<variant file="biped/gather_meat.xml"/>
-		<variant file="biped/rider/death_ranged.xml"/>
-	</group>
-	<material>player_trans.xml</material>
+    <variant file="biped/rider/cavalry/base_archer_fast_back.xml" name="Archer-Relax-Back" frequency="1"/>
+    <variant file="biped/rider/cavalry/attack_ranged_archer_back.xml"/>
+    <variant file="biped/rider/death_ranged.xml"/>
+  </group>
+  <material>player_trans.xml</material>
```
Comment (StanleySweet) [inline art/actors/units/byzantines/cavalry_archer_e_r.xml:42]: player_trans_spec see specmap above

File: art/actors/units/byzantines/cavalry_swordsman_c_r.xml [hunk 2]
```diff
@@ -117,16 +117,10 @@
       </textures>
     </variant>
   </group>
+
```
Comment (StanleySweet) [inline art/actors/units/byzantines/cavalry_swordsman_c_r.xml:120]: Remove

**Pull request merged**